.github/workflows/publish.yml

```diff
@@ -85,21 +70,15 @@ jobs:
           fi
       - name: Set build tag environment variable
         run: |
-          if [[ "${{ matrix.network }}" == "zen" ]]; then
-            echo "BUILD_TAGS=testnet netgo" >> $GITHUB_ENV
-            echo "ZIP_OUTPUT_SUFFIX=_zen" >> $GITHUB_ENV
-          else
-            echo "BUILD_TAGS=netgo" >> $GITHUB_ENV
-            echo "ZIP_OUTPUT_SUFFIX=" >> $GITHUB_ENV
-          fi
+          echo "BUILD_TAGS=netgo" >> $GITHUB_ENV
```

Comment:
This step seems superfluous now over setting the lone tag in the `go build` command